modules/oxide/src/less/theme/components/editor/editor.less

```diff
@@ -59,6 +59,14 @@
   }
 }
 
+// Remove styling if there is no Menubar of Toolbars in the header
+.tox.tox:not(.tox-tinymce-inline) .tox-editor-header.tox-editor-header--empty {
```

Comment:
Just checking if we need `.tox` to be specified twice?

Reply:
Yeah, it is for selector specificity’s sake, added a [clarifying comment](https://github.com/tinymce/tinymce/blob/915abeb35776729ea4d7dea253db1ba82e9128a6/modules/oxide/src/less/theme/components/editor/editor.less#L63) so it would not get accidentally patched out later on.

Reply:
Good to know. Thanks 👍